pkg/api/validation/validation_test.go

```diff
@@ -29,6 +29,9 @@ import (
 	utilerrors "github.com/GoogleCloudPlatform/kubernetes/pkg/util/errors"
 )
 
+const detailErrorFmt string = "error detail %s should be equal %s"
```

Comment:
I know this saves duplication, but it's duplication that doesn't matter - please inline these at call sites.  it's just more conventional Go.